data/hai/hai reveal 5 philosophers.txt

```diff
@@ -13,13 +13,14 @@
 
 mission "Hai Reveal [B01] Information Blackout"
 	landing
-	invisible
 	name "Information Blackout"
 	description "Meet the Syndicate Security Captain at the spaceport on <origin> when you're ready."
 	source "Sunracer"
 	clearance
 	to offer
 		has "Hai Reveal [A14] A Time of Trouble: done"
+	to complete
+		has "Hai Reveal [B02] Join the Fleet: offered"
```

Comment:
This whole thing is a no - what actually should happen here is that the conversation should end with a `decline` not an accept.

Reply:
You need to keep a reminder to arrive at the spaceport if the player decides to delay the mission.

Reply:
Why would you have a name and description if the mission is not to be shown, that's weird indeed

Reply:
I've removed those changes.